lib.php

```diff
@@ -25,7 +25,7 @@
 defined('MOODLE_INTERNAL') || die();
 
 require_once(__DIR__ . '/constants.php');
-
+require_once(__DIR__ . '/classes/task/adhoc_crawl_task.php');
```

Comment:
this shouldn't be needed if the class loading is setup right

Reply:
Not needed, have removed this.